### DISCUSSION TITLE
LuaSnip and autoclose interference

### DISCUSSION THREAD

**Discussion by aloispichler** [L3MON4D3/LuaSnip#1036] [Q&A]
Writing LaTeX I use snippets like
```
    "vector": {
        "prefix": ["\\begin{vector}" ],
        "body": ["\\begin{pmatrix} $1 \\\\\\\\ \\vdots \\\\\\\\ ${2:$1}\\end{pmatrix}$0"],
        "description": "vector"
    },
```
and I use autoclose as well to close `{` with  `}`.
Invoking the above snippet with `\begin{vec`, say, will give
```
\begin{pmatrix}  … \end{pmatrix}}
```
with one `}` too much at the end.
Grateful for any ideas how to prevent this behavior???

**Comment by L3MON4D3** [comment]
I personally don't use Autoclose, does it have some kind of temporary "disable"-switch?
Maybe call that before expand, and vim.schedule the reenable?
